mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/SendScreenV2.tsx

```diff
@@ -0,0 +1,611 @@
+                    await trigger('address')
+                  }
+                },
+                merge: true
+              })}
+              onClearButtonPress={async () => {
+                setValue('address', '')
+                await trigger('address')
+              }}
+              onAddressChange={async (address) => {
+                setValue('address', address, { shouldDirty: true })
+                await trigger('address')
+              }}
+            />
+            {matchedAddress !== undefined && (
+              <View style={tailwind('ml-5 my-2 items-center flex flex-row')}>
+                <ThemedIcon
+                  light={tailwind('text-success-500')}
+                  dark={tailwind('text-darksuccess-500')}
+                  iconType='MaterialIcons'
+                  name='check-circle'
+                  size={16}
+                />
+                <ThemedTextV2
+                  style={tailwind('text-xs mx-1 font-normal-v2')}
+                  light={tailwind('text-mono-light-v2-500')}
+                  dark={tailwind('text-mono-dark-v2-500')}
+                >
+                  {translate('screens/SendScreen', 'Verified')}
+                </ThemedTextV2>
+                <ThemedViewV2
+                  style={tailwind('flex flex-row items-center overflow-hidden rounded-full')}
+                  light={tailwind('bg-mono-light-v2-200')}
+                  dark={tailwind('bg-mono-dark-v2-200')}
+                >
+                  <View style={tailwind('rounded-l-2xl p-1.5')}>
+                    <RandomAvatar name={matchedAddress.address} size={16} />
+                  </View>
+                  <ThemedTextV2
+                    ellipsizeMode='middle'
+                    numberOfLines={1}
+                    style={[tailwind('text-xs font-normal-v2 mr-1'), {
+                      minWidth: 10,
+                      maxWidth: 108
+                    }]}
+                    light={tailwind('text-mono-light-v2-500')}
+                    dark={tailwind('text-mono-dark-v2-500')}
+                    testID='address_input_footer'
+                  >
+                    {matchedAddress.address}
+                  </ThemedTextV2>
+                </ThemedViewV2>
+              </View>
+            )}
+          </View>
+        )}
+
+        <View style={tailwind('mt-24 mx-12 items-center')}>
+          {(formState.isValid && token !== undefined) &&
+            <ThemedTextV2
+              testID='transaction_details_info_text'
+              light={tailwind('text-mono-light-v2-500')}
+              dark={tailwind('text-mono-dark-v2-500')}
+              style={tailwind('mt-2 text-xs text-center font-normal-v2')}
+            >
+              {isConversionRequired
+                ? translate('screens/SendScreen', 'By continuing, the required amount of DFI will be converted')
+                : translate('screens/SendScreen', 'Review full transaction details in the next screen')}
```

Comment:
figma and convert screen use a different msg, shall we follow?
```suggestion
                : translate('screens/SendScreen', 'Review full details in the next screen')}
```